pageserver/src/tenant/storage_layer/image_layer.rs

```diff
@@ -488,35 +490,33 @@ impl ImageLayerInner {
         let tree_reader =
             DiskBtreeReader::new(self.index_start_blk, self.index_root_blk, block_reader);
 
+        let btree_request_context = RequestContextBuilder::extend(ctx)
```

Comment:
Please shadow the `ctx`, less chance of mixing things up int he future


```suggestion
        let ctx = RequestContextBuilder::extend(ctx)
```

Reply:
https://github.com/neondatabase/neon/pull/7001/commits/9036ded6a1843fa6bb929da3bb153dfccd2a3255